.rubocop.yml

```diff
@@ -10,6 +10,10 @@ AllCops:
     - vendor/**/*
     - .git/**/*
 
+Metrics/MethodLength:
```

Comment:
why do you need to exlude this path?

Reply:
to allow create a table with a lot of fields

Reply:
i removed this